torch/jit/__init__.py

```diff
@@ -962,7 +964,13 @@ def _make_strong_submodule(field, module, parent):
         # It's not a submodule, don't do anything
         return None
 
-    return _convert_to_script_module(module)
+    # Convert the module to a ScriptModule
+    new_strong_submodule = _convert_to_script_module(module)
+
+    # Install the ScriptModule on the python side
+    parent._modules._python_modules[field] = new_strong_submodule
```

Comment:
Why did this change ?

Reply:
It will have a `nn.Sequential` until it is used and compiled to a `_ConstSequential` and installs that as a submodule of the `._c` `ScriptModule`. This keeps the Python `ScriptModule` in sync by adding it there